src/sentry_opsgenie/plugin.py

```diff
@@ -84,12 +84,7 @@ def build_payload(self, group, event):
             'entity': group.culprit,
         }
 
-        payload['details']['Event'] = dict(event.data or {})
```

Comment:
how come we no longer send this in details?

Reply:
It's rejected now since 'event' is not a string. Not sure if any of these fields are useful to people, I could individually add them back if so.

Reply:
oh ok I see.... hmmmmmmm, we still send like error message and stuff right? It seems fine to drop this for now to me

Reply:
Yep